ios/Podfile

```diff
@@ -1,9 +1,13 @@
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/Banuba/specs.git'
+source 'https://github.com/sdk-banuba/banuba-sdk-podspecs.git'
+
 # Uncomment this line to define a global platform for your project
-platform :ios, '12.1'
+platform :ios, '15.0'
```

Comment:
Why 15.0? Banuba SDK require 13.0 or higher

Reply:
[ve_sdk_flutter](https://github.com/Banuba/ve-sdk-flutter-integration-sample/blob/main/ios/Podfile#L5)

I see that it is 15.0 in the example project. could you share with me where they mention the 13.0 please?

Reply:
https://github.com/Banuba/ve-sdk-flutter/blob/master/README.md?plain=1#L46
https://github.com/Banuba/ve-sdk-flutter-integration-sample/blob/main/README.md?plain=1#L36

Reply:
i think documentation is old because i see these warnings

`[!] The platform of the target `Pods` (iOS 13.0) may not be compatible with `BNBLicenseUtils (1.39.0)` which has a minimum requirement of iOS 15.0.`

Reply:
Got it, resolved.